bool new_lp
int8 id
string tag
int8 map_id
int8 difficulty
int8 environment
int8 save_index
string database_path
